src/server/enterprise/cmds/cmds.go

```diff
@@ -99,29 +98,40 @@ func DeactivateCmd() *cobra.Command {
 
 // RegisterCmd returns a cobra.Command that registers this cluster with a remote Enterprise Server.
 func RegisterCmd() *cobra.Command {
-	var server, id string
+	var id, pachdAddr, enterpriseAddr string
```

Comment:
Where are these set?

Reply:
Nope, the flags are parsed automatically by cmdutils. We just define the variables and provide a pointer when defining the flags below.